src/relay/backend/aot_executor_codegen.cc

```diff
@@ -774,14 +697,17 @@ class AOTExecutorCodegen : public ExprVisitor {
       mod_run = pack_calls(mod_run);
     }
 
-    // Update the lowered functions
+    ret.function_metadata = std::move(function_metadata_);
```

Comment:
Is it possible for us to remove the specialized `main` functions `FunctionInfo` field in LoweredModule? They should all be uniformly set inside of the lowering, and I think the specialized `main` is a hold over from Lily's code that could be cleaned up.

Reply:
That field is still useful, and we can unify it between Graph and AOT to refactor altogether later; I've just pushed up a change to use the `main_func_info` attribute on the `LoweredModule` rather than the AOT bespoke way of processing this information so it can be refactored in unison later.